nx-X11/programs/Xserver/hw/nxagent/Options.h

```diff
@@ -384,6 +384,9 @@ typedef struct _AgentOptions
 
   /*
    * Inhibit some XKEYBOARD requests.
+   * 0 = allow all XKEYBOARD requests
+   * 1 = forbid all XKEYBOARD requests if keyboard=clone or keyboard=query
```

Comment:
Ah, and it makes sense: this the XKB extension is enabled when a client (re-)connects (and our first connection is also a reconnection for that matter), this will always be the case and disabling the XKB extension pretty much useless.